docs/md/docs/2020-07-09-last-dance.md

```diff
@@ -0,0 +1,96 @@
+There are a lot of disadvantages to this approach:
+* The team's ability to deliver is equal to the senior engineers' ability to deliver
+* The ability to improve is equal to the ability of your specific senior engineers to improve
+* This is not scalable and relies too much on too few people in your team
+* The non-senior team members do not improve, this approach perpetuates the current "snapshot" of the team - the seniors will stay seniors, the non-seniors will not be promoted.
+
+So even though the results were good, they were not great. Remember all the concerns team leads have on their heads? this approach led to unsatisfied internal stakeholders (not winning the championship), unstaisfied fans/customers (not winning the championship => the product quality was limited and did not improve), unsatisfied team (they were all a shadow of the senior engineer with no future to be seniors themselves), which makes the team lead itself unsatisfied - and who can blame them, how can you be satisfied when everyone around you are not?
+
+### Phil Jackson - win by maximizing the team performance
+Phil was hard to process for the team, he arrived with a completely different approach which basically turned upside down the way the team behaved so far. <br>
+The discussions and plans moved from "what can we do to make Michael Jordan score?" to "what can we do to win in the long term?". This is a huge difference that made the real change in the group. <br>
+Phil put emphasize on team building, made everyone focused on the target of winning and to rely on each other. "There is no I in team". <br><br>
+
+After some adaption time there was a huge difference in the team performace that led to winning the championship. All the disadvantages from before disappeared:
+* The team's ability to deliver is equal to the entire team's ability to deliver
+* The ability to improve is equal to the entire team's ability to improve
+* This is as scalable as it can get - it reliaes on everyone in the team
+* The non-senior team members improve because the senior team members push them to excellence
+
+This approach makes the stakeholders, fans/customers and the team satisfied.<br>
+When you put weight on the long term goal and make sure everyone on the team improves you get the best outcomes. Focusing on one person or on short term goals (winning only the next game and not the next seasons) don't get you to your maximum.
+
+![Michael Jordan and Dennis Rodman](/docs/assets/last-dance/mj-and-dennis-rodman.jpeg)
+
+## The role of the senior team member
+Doug Collins believed that the senior engineer's role is to bring success by leading the team and carry it forward.<br>
+Phil Jackson believed that the senior engineer's role is to bring success by pushing the team to excellence "from behind", being a servant leader that defines success by the team's success and not by personal success.
```

Comment:
when writing "from behind" do you mean from "behind the scenes"?

Reply:
nope, meant to "push" them from behind, encouraging them to improve and not doing the job for them, as explained later in the sentence